BRUSdevice/Logical/Lib/BRUSdevice/CIFSDevChk.st

```diff
@@ -133,11 +129,6 @@ FUNCTION_BLOCK CIFSDevChk
 		// call device related functions
 		IS.fb.devLink();
 		IS.fb.devUnlink();
-				
-		// check for update commands
-		IF Update AND NOT IS.old.update THEN
-			IS.stat.initDone		:= FALSE;
-		END_IF
```

Comment:
🟢 removing this ensures the Update command only works in idle
❔ maybe we can still take the update request but populate Info.Status to say that you are only allowed to update when there are no devices linked?